### PR TITLE
Update manual test plan for partners

### PR DESCRIPTION
**Pull request by lukewrites**
First attempt at making manual test passes less onerous for partners.

Removed:
- "Verify that you can duplicate the data source" step
- `Verify that the data source connection properties are correct.`
- "Edit your connection" step - they may not have multiple endpoints to test against.
- "Missing connector" steps

Added:
- connecting with each supported auth mode

### CODE REVIEW

File: docs/manual-test.md [hunk 1]
```diff
@@ -97,59 +89,6 @@ Right-click the data source, then click **Extract Data**. Verify that you can cr
...
-
-1. Right-click the data source and click **Edit Data Source**.<br/>
-![]({{ site.baseurl }}/assets/mt-edit-data-source.png)
-The worksheet opens in Tableau.
-1. In the left pane, under **Connections**, click the dropdown menu next to the server name and click **Edit Connection**.<br/>
-![]({{ site.baseurl }}/assets/mt-edit-connection.png)
-1. Change something. For example, change the server.<br/>
-![]({{ site.baseurl }}/assets/mt-change-server.png)
-1. After you click **Sign In**, you should see the new server name under **Connections**.<br/>
-![]({{ site.baseurl }}/assets/mt-new-server.png)
```
Comment (pvanderknyff) [inline docs/manual-test.md:112]: I think this should be kept, edit connection isn't that onerous and I think there are some scenarios where the connector changes can break it.

**Review by pvanderknyff — COMMENTED** [review]
Looks a lot better! Have one test case I think should be kept, the rest looks good to me.

File: docs/manual-test.md (hunk 1 above)
Comment (lukewrites) [reply docs/manual-test.md:112]: Sounds good. Are there options other than "change the server" we could suggest?

Comment (lukewrites) [reply docs/manual-test.md]: reverted

Comment (pvanderknyff) [reply docs/manual-test.md]: Could change the database, or the user account, or auth method. Any of the fields should suffice.

**Review by pvanderknyff — APPROVED** [review]

**Pull request merged**